configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -1168,47 +1180,48 @@
         </div>
     </section>
 
-    <section id="transfer-out-section" sectionTag="fieldset"
-             headerTag="legend" headerStyle="title" headerCode="pihcore.transfer.title">
+    <section id="transferIn" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+             headerCode="pihcore.transferReferralIn">
         <div class="section-container-color">
-            <obsgroup groupingConceptId="PIH:13170">
+            <obsgroup groupingConceptId="PIH:13169">
                 <label>
-                    <uimessage code="pihcore.externalTransferQuestion" />
+                    <uimessage code="pihcore.externalTransferQuestion"/>
                 </label>
-                <p class="radio cb-align">
-                    <obs conceptId="PIH:HUM Disposition categories"
-                         answerConceptId="CIEL:160036" answerCode="emr.yes"
-                         style="checkbox" toggle="transfer-out-details"/>
```

Comment:
I was thinking we would keep this question and just add in the second question, "referral or transfer" but you are right, it might be cleaning just to replace the questions... we will have to update the migration code to migration this information properly, however.

I don't know if a radio button will work though, because I think with a radio button once a user clicks on option they can't  unclick the question entirely... but maybe I'm wrong about that.

Thoughts?